src/liquid/place/bulk_create_form.html

```diff
@@ -4,8 +4,9 @@
     hx-encoding='multipart/form-data' 
     autocomplete="off"
     hx-post="/place?op=bulk&type={{contactType.name}}"
-    hx-target="this" 
-    hx-swap="outerHTML"
+    hx-disabled-elt="button#place_create_submit"
```

Comment:
i like the use of `hx-disable-alt` why do we need to do all the other stuff? are you trying to hide it?
is disabling the button not enough?

Reply:
After uploading the csv there's a period of time, when fetching the next page, that the buttons are still visible. You can still click the `Add` button then. I think i can just use the progress bar here from the start and not use `hx-disabled-elt`

Reply:
Is `hx-indicator` useful?
Could we maybe use an animated spinner (https://github.com/medic/cht-user-management/issues/62) instead making them empty?

Reply:
Maybe not, the indicator will be visible for the duration of the csv upload request. When that requests completes there's still the transition period between the upload form and the next page where a user could still click the `Add` button

Reply:
Is this still a problem now that we lazy-load the data?

Reply:
I also think It's safe to ignore now, the user would need a really bad connection or sub-100ms reaction time to view/click that button